srcpkgs/brother-mfcj6720dw-cupswrapper/files/cupswrapper.void

```diff
@@ -0,0 +1,283 @@
+#! /bin/sh
+#
+# Brother Print filter
+# Copyright (C) 2005-2011 Brother. Industries, Ltd.
+
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation; either version 2 of the License, or (at your option)
+# any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+# Place, Suite 330, Boston, MA  02111-1307  USA
+#
+# modified for voidlinux by Carlo Dormeletti carlo.dormeletti <at> email.it
+# 2014/04/26 -- first version
+# 2014/08/25 -- adapted to runit
+#
+
+
+printer_model=""mfcj6720dw""
+cups_printer_name=`echo $printer_model | tr '[a-z]' '[A-Z]'`
+device_name=`echo $cups_printer_name | eval sed -e 's/MFC/MFC-/' -e 's/DCP/DCP-/' -e 's/FAX/FAX-/'`
+# pcfilename=`echo $cups_printer_name | tr -d '[A-Z]'`
+device_model="Printers"
+tmp_filter=/var/tmp/brlpdwrapper${printer_model}
+ppd_file_name=/usr/share/ppd/cupsfilters/brother_${printer_model}_printer_en.ppd
+brotherlpdwrapper=/usr/lib/cups/filter/brother_lpdwrapper_${printer_model}
```

Comment:
In `brother-dcp197c-cupswrapper`, the `$brotherlpdwrapper` is called `brlpdwrapper${printer_model}` but I noticed that prevented my printer from working. In the logs, I found cups crying about a missing `brother_lpdwrapper_mfcj6720dw` but I have no idea what convinced it it was at that location and not `brlpdwrappermfcj6720dw` and since I find it clearer with the `_` I thought I'd rename it here. If someone knows how to change the name expected by cups and if it's something really important not to mess with, I'll revert this line to what it was in `brother-dcp197c-cupswrapper`.